### ISSUE TITLE
Flip Cloud Foundry deploy into production mode

### ISSUE THREAD

**Issue by konklone** [18F/foia-hub#617]
It's been running in the development environment, but should be flipped to the production environment. The production environment will need to add more hostnames to `ALLOWED_HOSTS` in `foia_hub/settings/production.py`.

To flip the app to production:

```
cf set-env foia-a DJANGO_SETTINGS_MODULE foia_hub.settings.production
cf set-env foia-b DJANGO_SETTINGS_MODULE foia_hub.settings.production
```

Then redeploy:

```
cf restage foia-a
cf restage foia-b
```

**Comment by konklone** [comment]
A few notes here:

* Now that the app forces HTTPS in production after #619, this ticket depends on the verification of HTTPS, DNS, and ELB configuration in #553 before it's safe to enable for `foia-a` or `foia-b`.
* Once enabled, `foia-a.cf.18f.us` and `foia-b.cf.18f.us` will no longer be viable URLs to use to inspect the state of the production system -- only `https://open.foia.gov` will work (and when the A record is removed pre-launch, `/etc/hosts` will be necessary to access the production system).
* DevOps plans to change the base domain we run CloudFoundry at to get it off of the `cf.18f.us` base domain and onto a `.gov` itself. This is also when it makes sense to finally purchase a wildcard cert, so that dev/staging domains can safely and easily be on HTTPS without any work on the part of individual project devs.

In the meantime, our CloudFoundry staging domain (`foia.cf.18f.us`?) will continue to be fine to be accessed over `http://` if it remains in development mode.

In other words: **don't run the above command and close this ticket until #553 is closed.**

**Comment by khandelwal** [comment]
As discussed before - I set up a staging environment last week. It runs here: http://openfoia-staging.cf.18f.us/ (no https setup).

**Comment by konklone** [comment]
The production app(s), `foia-a` and `foia-b` are now in production mode and have been re-deployed. I updated the commands at the top to reflect the process I went through.
